### DISCUSSION TITLE
Set up and usage

### DISCUSSION THREAD

**Discussion by Yomuddas** [jef/zap2xml#6] [Q&A] [answered]
Sorry for bothering but I seem to be at a dead stop I have no clue how to set this up and I've been trying to get this set up or just about a week. I would like it to write the file to a certain directory on my external hard drive. but I cant seem to even get that far with it. If someone, anyone can run me through a set up for this on windows 10 I'll be truly grateful. I have Perl and docker installed including ubunto for wsl.

**Comment by jef** [comment]
I'm not sure how WSL and Docker work together, but take a look online about volume mounting and that may help you.

**Comment by jef** [comment, marked as answer]
Take a look at the new code base. You may find it easier to setup now.

**Reply by jef** [reply]
https://github.com/jef/zap2xml/wiki
